packages/env/package.json

```diff
@@ -33,6 +33,7 @@
 	},
 	"dependencies": {
 		"chalk": "^2.4.2",
+		"copy-dir": "^1.2.0",
```

Comment:
We could potentially avoid pulling in a new dependency by reusing `fs-extra` (already a `devDependencies` in the root `package.json`), which is also promises-based (avoids need to promisify).

https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/copy.md

Reply:
I started that way, but opted to optimize for a leaner `@wordpress/env`. `fs-extra` is 10x bigger.

Reply:
> I started that way, but opted to optimize for a leaner `@wordpress/env`. `fs-extra` is 10x bigger.

That's fair. It's not as optimal for _our_ project to have multiple dependencies capable of achieving the same goal, but good to be thinking about the consumer here. Might be a good reason to look to whether we should try to replace or remove our existing usage of `fs-extra`, which I know to be quite limited.

In fact, I think the only thing we really "need" it for is this directory removal:

https://github.com/WordPress/gutenberg/blob/598211ef1dcf9f08688d3de446b773c8f6979f12/bin/commander.js#L246-L247

Historically I might have used [`rimraf`](https://www.npmjs.com/package/rimraf) for this.

Apparently `recursive` is available as an experimental option now in the latest Node LTS, so we could consider that option as well: https://nodejs.org/dist/latest-v12.x/docs/api/fs.html#fs_fs_rmdir_path_options_callback

Reply:
>Historically I might have used rimraf for this.

And we already have `rimraf`. I'll make a note to replace it.

>Apparently recursive is available as an experimental option now in the latest Node LTS, so we could consider that option as well

When it becomes stable right?

Reply:
> > Apparently recursive is available as an experimental option now in the latest Node LTS, so we could consider that option as well
> 
> When it becomes stable right?

Ehh, it doesn't bother me much. Well.. assuming that it's experimental in the sense of "The API signature may change in the future" and not "Your entire filesystem might be destroyed".

Reply:
>Stability: 1 - Experimental. The feature is not subject to Semantic Versioning rules. Non-backward compatible changes or removal may occur in any future release. Use of the feature is not recommended in production environments.

Reply:
#18790